consensus/src/consensus.rs

```diff
@@ -490,16 +543,6 @@ impl<R: ConsensusRpc> ConsensusStateManager<R> {
             return Err(ConsensusError::InvalidCurrentSyncCommitteeProof.into());
         }
 
-        self.last_checkpoint = Some(checkpoint.to_vec());
-        self.store = LightClientStore {
-            finalized_header: bootstrap.header.clone(),
-            current_sync_committee: bootstrap.current_sync_committee,
-            next_sync_committee: None,
-            optimistic_header: bootstrap.header.clone(),
-            previous_max_active_participants: 0,
-            current_max_active_participants: 0,
-        };
-
```

Comment:
I think this could have stayed here no?

Reply:
it's only red here because of the order of the code, this is not really removed from logic.